apache2/apache2_util.c

```diff
@@ -344,6 +344,14 @@ static void send_waf_log(const char* timestamp, struct waf_lock* lock, apr_file_
     get_short_filename(waf_filename);
     get_ruleset_type_version(waf_ruleset_info, waf_ruleset_type, waf_ruleset_version); 
 
+    // Format UTC time
+    time_t rawtime;
+    struct tm * timeinfo;
+    char timestamp[30];
+    time(&rawtime);
+    timeinfo = gmtime(&rawtime);
+    strftime(timestamp, 30, "%Y-%m-%dT%H:%M:%S+00:00", timeinfo);
```

Comment:
1) Either give 30 some name through #define or static const or use `sizeof(timestamp)` here. This will make sure we always use the same value in place of buffer declaration and formatting - if we ever modify it we'll only need to modify it in one place
2)

Reply:
Good idea